### DISCUSSION TITLE
Failed to find index datastore at path solution?

### DISCUSSION THREAD

**Discussion by jamesdabrowski-ast** [peripheryapp/periphery#585] [Ideas]
While trying to use periphery I kept running into the above error: Failed to find index datastore at path: [insert path here]. I think I figured out what was causing that issue for me. In my project I have 1 scheme, that builds 1 target (both of the same name) but also have 4 different .xcconfig files I use to build different variants of that app. I had the scheme set to use a config file that produced an app with a different name than that of the scheme and target and I kept getting the above error. When I changed the scheme setting to use the config that produced an app with the same name as the scheme and target, periphery scanned and worked just fine. I'm not sure if this is a new feature request or just a point of interest - maybe just add something to the README addressing this? 
Great tool. Thanks for building and maintaining this for us. Appreciate it.

**Comment by azurast** [comment]
facing the same issue
